spec/system/home_spec.rb

```diff
@@ -143,6 +143,34 @@
     expect(page).not_to have_css(".title", text: "Featured")
   end
 
+  describe "Cards are ordered" do
+    scenario "by order field" do
+      create(:widget_card, title: "Card one", link_text: "Link one", order: 2)
+      create(:widget_card, title: "Card two", link_text: "Link two", order: 1)
+      create(:widget_card, title: "Card three", link_text: "Link three", order: 0)
```

Comment:
Since we aren't usingthe `link_text` field, maybe we can simplify the test. Also note the test would pass if cards were displayed in creation order, with the last card appearing first, so maybe we can change the orders to something like:

```suggestion
      create(:widget_card, title: "Card one", order: 3)
      create(:widget_card, title: "Card two", order: 1)
      create(:widget_card, title: "Card three", order: 2)
```